env/sample.properties

```diff
@@ -10,6 +10,7 @@ STATSD_ENABLED=false
 STATSD_HOST=localhost
 STATSD_PORT=8125
 STATSD_PREFIX=beast
+STATSD_TAGS=
```

Comment:
Since default value is empty string, we can remove this from sample.properties

Reply:
I think we need this. Systems using beast will be aware of this property.